src/pages/office/office.ts

```diff
@@ -235,8 +249,35 @@ export class OfficePage extends BasePageComponent {
         select(getTestData),
         map(data => getSeriousOrDangerousFaults(data.dangerousFaults)),
       ),
+      drivingFaults$: this.store$.pipe(
```

Comment:
Can you please extract the `testData` into it's own observable and reuse it in these different selections?

Reply:
Done